src/main/flight/pid.c

```diff
@@ -267,6 +267,17 @@ void resetPidProfile(pidProfile_t *pidProfile)
         .chirp_frequency_start_deci_hz = 2,
         .chirp_frequency_end_deci_hz = 6000,
         .chirp_time_seconds = 20,
+#if defined(USE_WING)
```

Comment:
Being uniform with macro usage in other places
```suggestion
#ifdef USE_WING
```